src/app/core/notifications/notifications.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable, NgZone } from '@angular/core';
+import { NotificationStyles } from './notification-styles';
+import { MatSnackBar, MatSnackBarConfig } from '@angular/material';
+
+/** UI notifications service */
+@Injectable()
+export class NotificationsService {
+  constructor(
+    private readonly snackBar: MatSnackBar,
+    private readonly zone: NgZone
+  ) {}
+
+  /** Displays a self-closing UI notification with a given message.
+   * @param {string} message - Notification message.
+   * @param {NotificationStyles=} style - Notification style, Information by default.
+   */
+  show(message: string, style?: NotificationStyles) {
```

Comment:
Hi @hhubik !

Thanks for your PR! What is your opinion about providing `.info()`, `.warn()`, `.error()`, `.success()`, ... etc convenience methods instead of passing `enum` value ?

Reply:
Hi @tomastrajan I don't have a strong preference one way or another.  I actually briefly considered doing what you suggest, but then decided to do it this way, probably because in my previous life I did C#.  For example, to show a message dialog in C#, you call `MessageBox.Show()` method and control the appearance and content of the message box with input parameters to `Show()`.

If the Javascript/Typescript convention is to use separate methods, I am happy to do that.  Just let me know.  

In general, I prefer to use verbs in methods names.  So I would probably do `.showInfo()` etc.  Is that OK?

Reply:
@hhubik thank you for your explanation. The way I think about the notifications is like a more fancy console. That means it kinda follow console API like info, warn, error, but also adds more user specific stuff like `success`. In terms of styling it could be something like this...

info: $primary color from material theme
error: $warn from material ( don't remember the exact name)
success: we have to define another variable for this ( pastel green )
warn: we have to define another variable for this ( pastel yellow )

we should define these new variables globalin `styles-variables.scss` and import wherever needed ( in the `-scss-theme.scss` files.

Please let me know if you have more questions ;)

Reply:
@tomastrajan - I have found a few examples of notification services for Web UI.  They all use your approach.  I will change my code to match that.  Before I do that though, would it make more sense to just use an off-the-shelf solution such as [angular2-notifications](https://www.npmjs.com/package/angular2-notifications)?

Reply:
@hhubik I would go with our own lightweight notification service :)

Reply:
@tomastrajan OK.  I am making the changes to the code now.  Could you please elaborate on "we have to define another variable for this ( pastel green )".  Is there an example of how this should be done? Thanks!

Reply:
@hhubik so for the success and warning notifications, you will need to define color which are cannot be defined as a part of the theme so you have to define them. 

Themes doesn't support hanging additional properties so for now, we have to define these colors globally in the `styles-variables.scss`. 

They can be called `$success: mat-palette($mat-light-green);` and `$warning: mat-palette($mat-yellow, 400);`....

Then you can use them in the component where you implement notifications (preferably a new component) which will be used in the `app.component.html` template. In terms of applying color, please use additional theme file as for every component (all of them have these files: ts, html, scss and scss-theme.scss)